apply.md

```diff
@@ -0,0 +1,23 @@
+# How to apply to WiMLDS Berlin Data Science Lab?
+
+We are currently taking application for a new round of the lab to start in March 2021!
+
+## Prerequisites
+
+1. Must have some theoretical data science knowledge, i.e. taken some in-person/online classes or learned from books. You do not need to be an expert in data science, but you should be familiar with concepts such as feature engineering, classification vs regression models, train/test splitting, evaluation metrics.  
+2. Must have some experience using R or python/scikit-learn for data science.
+3. Recommended to have some familiarity with Git (for collaboration and showcasing your work), but it is not required to use GitHub to complete this project
+
+## Time requirements
+
+You should be available for two 1-hour calls/week over the course of 10 weeks. We will schedule these calls before the lab starts, so that you can plan ahead. 
+
+In addition, you should expect 5-10 hours/week for working on the project (no fixed times, however fits your schedule).
```

Comment:
5-10 or more?

Reply:
Fair warning, some of our previous participants chose to invest more than 8 hours/week simply because they found the project engaging and wanted to progress further with their ideas.